### DISCUSSION TITLE
Home Module

### DISCUSSION THREAD

**Discussion by Yertled** [Zrips/CMI#7309] [Enhancement]
Being able to completely disable the /home system in CMI.
Context: I am using a hybrid forge / paper server like Magma, and CMI is buggy with it, so people setting homes in other dimensions don't get teleported there, rather the coords in overworld.

**Comment by mrfloris** [comment]
alias.yml, set home related commands to false, then in permission manager, negate the permissions for these commands, and in modules.yml, disable home interactions, and in config.yml search for home settings and configure them to your liking.
